scalameta/dialects/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -88,17 +94,31 @@ package object dialects {
     def name = "Scala211"
     def bindToSeqWildcardDesignator = Scala210.bindToSeqWildcardDesignator
     def allowXmlLiterals = Scala210.allowXmlLiterals
+    def allowInline = false
     def allowSpliceUnderscore = true // SI-7715, only fixed in 2.11.0-M5
     def allowToplevelTerms = Scala210.allowToplevelTerms
     def toplevelSeparator = Scala210.toplevelSeparator
     def metalevel = Metalevel.Zero
     private def writeReplace(): AnyRef = new Dialect.SerializationProxy(this)
   }
 
+  @leaf implicit object Paradise extends Dialect {
```

Comment:
I didn't want to overengineer this, since for the time being 2.11 is going to be the only supported Scala version for Paradise 3.x. But migration is actually a valid concern. I'll rename this to `Paradise211` now.